qiskit_dynamics/models/generator_model.py

```diff
@@ -529,3 +420,141 @@ def evaluate_rhs(self, time: float, y: Array, in_frame_basis: Optional[bool] = F
             return op_combo @ y
 
         return out
+
+    @classmethod
+    def transfer_static_operator_between_frames(
```

Comment:
Is there a particular reason this and the other transfer operators func are class methods of this class rather than just a function you use directly? They don't seem to make any use of the class instance.